src/coreclr/vm/ceemain.cpp

```diff
@@ -755,6 +755,13 @@ void EEStartupHelper()
 #endif // !TARGET_UNIX
         InitEventStore();
 
+#ifdef TARGET_AMD64
+        // We only do this on amd64 (NOT ARM, because ARM uses frame based stack crawling),
```

Comment:
We don't do this on non-Windows either, but the ifdefs are in different spot. I think it would look best to have a no-op implementation of `UnwindInfoTable` for non-Amd64 non-Windows so that these ifdefs are not spread around.